src/data/strings/jsonStringsBackend.js

```diff
@@ -118,23 +86,33 @@ export default class JSONStringsBackend extends StringsBackend {
   }
 
   resolveContext(context) {
-    if (context === this.getDefaultContext()) {
-      return HomeJSON;
+    const splitArr = context.split('.');
+    if (context === 'assets') {
+      return axios.get(`${base}global/`)
+        .then(response => {
+          JSONMap[context] = response.data;
+        });
+    } else if (splitArr.length > 1 && splitArr[0] === 'projects') {
+      return axios.get(`${base}project/${splitArr[1]}`)
```

Comment:
Use Axios Instances instead of string templates...
https://github.com/axios/axios